lib/dockerize_rails/docker_commands/helpers.rb

```diff
@@ -0,0 +1,33 @@
+module DockerizeRails
+  module DockerCommands
+    module Helpers
+      NAMES = {
+        image: {
+          rails: "#{DRConfig.application_name}_rails:#{DRConfig.application_env}".freeze,
+          mysql: "#{DRConfig.application_name}_mysql:#{DRConfig.application_env}".freeze,
+          postgres: "#{DRConfig.application_name}_postgres:#{DRConfig.application_env}".freeze
+        }.freeze,
+        container: {
+          rails: "#{DRConfig.application_name}_rails_container".freeze,
+          mysql: "#{DRConfig.application_name}_mysql_container".freeze,
+          postgres: "#{DRConfig.application_name}_postgres_container".freeze
+        }.freeze
+      }.freeze
+
+      def self.get_name(service, type)
+        NAMES[type][service]
+      end
+
+      def self.services_from_docker_compose
+        docker_compose = YAML.load_file "#{DockerizeRails::PATHS.current}/docker-compose.yml"
+        docker_compose['services']
+      end
+
+      def self.recurse_merge(a, b)
```

Comment:
__[reek]__
* [UncommunicativeParameterName] has the parameter name 'a' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)
* [UncommunicativeParameterName] has the parameter name 'b' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Parameter-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->